### PR TITLE
Update export and import search setting about a known issue.

### PR DESCRIPTION
**Pull request by DanteMustDie**
We migrate search setting to 3S and a compatibility issue is introduces that customer could not import search setting that exported before migration into 3S.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointOnline/export-and-import-search-settings.md [hunk 1]
```diff
@@ -196,8 +196,16 @@ If managed properties or aliases contain any of the listed characters, the impor
...
 
-Since you are unable to check the status of the search configuration file you imported from the UI, you could choose an alternative way to access the list, like [SharePoint CSOM API](/sharepoint/dev/sp-add-ins/complete-basic-operations-using-sharepoint-client-library-code) or [SharePoint Online REST API](/sharepoint/dev/sp-add-ins/complete-basic-operations-using-sharepoint-rest-endpoints).
\ No newline at end of file
+Since you are unable to check the status of the search configuration file you imported from the UI, you could choose an alternative way to access the list, like [SharePoint CSOM API](/sharepoint/dev/sp-add-ins/complete-basic-operations-using-sharepoint-client-library-code) or [SharePoint Online REST API](/sharepoint/dev/sp-add-ins/complete-basic-operations-using-sharepoint-rest-endpoints).
+
+ **Unable to export stale search configuration xml file**
+
+If the search configuration file is exported long time ago, then the import might fail.
+
+If you are improting a stale exported search configuration file, please export a new search configuration firstly and then use it to import.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/export-and-import-search-settings.md:211]:
```suggestion
 **Unable to export stale search configuration XML file**

If the search configuration file was exported a long time ago, the import might fail.

If the exported search configuration file is stale, you should export a new search configuration first and then use it to import the list of search configuration files.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @DanteMustDie, kindly commit the suggested changes if you agree with them. Many thanks.

**Comment by DanteMustDie** [comment]
> Copy review - @DanteMustDie, kindly commit the suggested changes if you agree with them. Many thanks.

**Comment by DanteMustDie** [comment]
Sorry for error operation. @JohanFreelancer9 already accepted your suggestion, thanks for review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@SerdarSoysal PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**